src/rust/wpcomics/sources/nettruyen/res/source.json

```
@@ -18,4 +18,4 @@
 			"name": "Truyện con trai"
 		}
 	]
-}
+}
```

don't remove the trailing newline
```suggestion
}

```